openapi/components/schemas/OrderItem.yaml

```
@@ -18,6 +18,9 @@ properties:
   quantity:
     description: Number of product units in the specified plan.
     type: integer
+  quantityUsed:
```

I wonder to use the term "Used" or "Filled"?

```suggestion
  quantityFilled:
```

---

Good point! 'Used' is ambiguous here. I think the term 'filled' more closely relates to the number of items in the order, and these items may not be 'used' yet.

We should also update [line 22](https://github.com/Rebilly/api-definitions/pull/1754/files#diff-2f5d627238c8032bc7c2dcf9595f83f6cad042a6c8119b492ba09d9642f3ccdbR22) to `Number of filled product units.`